src/server/internalServer.cpp

```diff
@@ -187,6 +189,9 @@ bool InternalServer::start() {
   auto server_start_time = std::chrono::system_clock::now().time_since_epoch();
   m_server_id = kiwix::to_string(server_start_time.count());
   m_library_id = m_server_id;
+  std::string url = "http://" + m_addr + ":" + std::to_string(m_port) + "/";
+  std::cout << "The Kiwix server is running and can be accessed in the local network at: "
+            << url << std::endl;
```

Comment:
We must not print the message from libkiwix:
- It is to kiwix-serve to print the message to the user.
- It would not be usefull when the server is run from kiwix-desktop as user doesn't have access to the console output.

We need to define a new method to get the url and make the tools using `libkiwix`'s server display the information to the user.

Reply:
So, 
I should remove this helper line.
make a method to get URL. 
And use that URL to implement https://github.com/kiwix/kiwix-tools/issues/473 separately

Reply:
Yes. And also do the same in kiwix-desktop (we already display a IP address but we should use the new method in libkiwix)